Formula/portable-ruby.rb

```diff
@@ -47,6 +48,13 @@ class PortableRuby < PortableFormula
     end
   end
 
+  # Fix rare hang on forking. Affected Homebrew/core CI, especially on macOS 14 x86_64.
+  # Patch in production use at Stripe.
```

Comment:
Could this get link(s) to PRs/issues? thanks!

Reply:
https://github.com/Homebrew/homebrew-portable-ruby/pull/231